DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -42,8 +42,10 @@ constexpr int NCOLMCM = 18;     // the number of pads per MCM
 
 constexpr int NBITSTRKLPOS = 11;                   // number of bits for position in tracklet64 word
 constexpr int NBITSTRKLSLOPE = 8;                  // number of bits for slope in tracklet64 word
-constexpr float GRANULARITYTRKLPOS = 1.f / 75;     // granularity of position in tracklet64 word in pad-widths
-constexpr float GRANULARITYTRKLSLOPE = 1.f / 1000; // granularity of slope in tracklet64 word in pads/timebin
+constexpr float PADGRANULARITYTRKLPOS = 80.f;
+constexpr float PADGRANULARITYTRKLSLOPE = 1000.f;
+constexpr float GRANULARITYTRKLPOS = 1.f / PADGRANULARITYTRKLPOS;     // granularity of position in tracklet64 word in pad-widths
+constexpr float GRANULARITYTRKLSLOPE = 1.f / PADGRANULARITYTRKLSLOPE; // granularity of slope in tracklet64 word in pads/timebin
```

Comment:
Do we want these values hardcoded, or is there a chance that we will change them at a later stage? Last week it looked like the ranges are too big, and that we should rather increase the resolution. Will this adjustment be done once, or do we foresee further tuning in the future?